semgrep-core/src/engine/Match_tainting_mode.ml

```diff
@@ -495,22 +495,31 @@ let check_fundef lang options taint_config opt_ent fdef =
     let* name = AST_to_IL.name_of_entity ent in
     Some (D.str_of_name name)
   in
-  let add_to_env env id ii =
+  let add_to_env env id ii pdefault =
+    pr2 (spf "add_to_env %s pdefault?=%b" (fst id) (Option.is_some pdefault));
```

Comment:
remaining pr2 ...
Maybe we need to add a semgrep rule to forbid them.

Reply:
use logger#debug if you really need.

Reply:
Ooops, sorry, that's meant to go away. Good catch.